app/src/main/java/com/riteh/autoshare/ui/user/CardNumberFormatX.kt

```diff
@@ -14,6 +14,9 @@ class CardNumberFormatX: TextWatcher {
     override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {
     }
 
+    /**
+     * Put char '-' after four digits
+     */
```

Comment:
Nije samo za to; ta funkcija ignorira non-digit znakove, umeće "-", pazi na duljinu itd. U biti možeš napisati da formatira upisani tekst.